airbyte-webapp/src/utils/errorStatusMessage.tsx

```diff
@@ -1,6 +1,10 @@
 import { FormattedMessage } from "react-intl";
 
-export const createFormErrorMessage = (error: { status?: number; message?: string }): JSX.Element | string | null => {
+export class FormError extends Error {
+  status?: number;
+}
+
+export const createFormErrorMessage = (error: FormError): JSX.Element | string | null => {
```

Comment:
Why is this specific to `createForm`?  This looks more generic than that.  I'm also not sure from the name if this means there was an error creating the form or an error within the creation form (for connections? for connectors?)

Reply:
Maybe it's for creating a form error message!  Again, this seems more generic than forms?  and maybe `parseErrorMessage` would be less ambiguous?

Reply:
`generateMessageFromError`